app/common/view_s.c

```diff
@@ -50,9 +51,25 @@ static void h_account_toggle();
 static void h_account_update();
 #endif
 
+#ifdef APP_BLIND_MODE_ENABLED
+static void h_blind_toggle();
+static void h_blind_update();
+#endif
+
+enum MAINMENU_SCREENS {
+    SCREEN_HOME = 0,
+    SCREEN_EXPERT,
+#ifdef APP_ACCOUNT_MODE_ENABLED
+    SCREEN_ACCOUNT,
+#endif
+#ifdef APP_BLIND_MODE_ENABLED
+    SCREEN_BLIND,
+#endif
+};
```

Comment:
This enum is only used internally for nanoS UI. I created this enum like that because I needed a dynamic enumeration depending on which modes/features were enabled.
Just to clarify, the callback function will be called with 0x02 (from sdk) and that might have a different meaning for every app.